src/number_theory/pythagorean_triples.lean

```diff
@@ -534,6 +534,60 @@ begin
     { split, { ring }, rw int.gcd_comm, exact coprime_pow_two_sub_mul co pp } }
 end
 
+private lemma neg_parity_eq_parity (i : ℤ) : (-i) % 2 = i % 2 :=
```

Comment:
I think this should move to some basic file about int, and it shouldn't be a private lemma